packages/yew-bootstrap/src/component/pagination.rs

```diff
@@ -0,0 +1,95 @@
+use yew::prelude::*;
+
+
+#[derive(Clone, PartialEq, Eq)]
+pub enum PaginationSize {
+    Large,
+    Normal,
+    Small,
+}
+
+impl Default for PaginationSize {
+    fn default() -> Self {
+        PaginationSize::Normal
+    }
+}
+
+pub struct Pagination {}
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    #[prop_or_default]
+    pub callback: Callback<u32>,
+
+    #[prop_or_default]
+    pub current_page: u32,
+
+    #[prop_or_default]
+    pub max_items_per_page: u32,
```

Comment:
It seems items per page is beyond the scope of this component. It doesn't maintain any list state at all, so the max items is outside of its scope. It only needs to know how many pages and the current page. Everything beyond that is rendering and interaction logic.